pkg/expr/commands.go

```diff
@@ -184,11 +185,12 @@ func (gr *ReduceCommand) Execute(ctx context.Context, _ time.Time, vars mathexp.
 			}
 			copyV := mathexp.NewNumber(gr.refID, v.GetLabels())
 			copyV.SetValue(value)
-			if gr.seriesMapper == nil {
+			if gr.seriesMapper == nil && !noticeAdded { // Add notice to only the first result to not multiple them in presentation
```

Comment:
Since in a loop, set `i` (or whatever above), and then the check can just be `&& i == 0`